TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -99,38 +113,59 @@ public LaneConnectorTool(TrafficManagerTool mainTool)
             Vector3 camPos = Singleton<SimulationManager>.instance.m_simulationView.m_position;
 
             // Bounds bounds = new Bounds(Vector3.zero, Vector3.one);
-            Ray mouseRay = Camera.main.ScreenPointToRay(Input.mousePosition);
+            Camera currentCamera = Camera.main;
```

Comment:
Cache camera reference before using. It can boost performance too

Reply:
This is not in a loop, so probably not worth doing it.

Reply:
Of course it is ;) `ShowOverlay()` is called once per frame for each tool (with overlay defined)

Reply:
This takes __0.005ms__ 

Not bad, I think we can ignore that for now